templates/common/package.json

```diff
@@ -50,7 +51,8 @@
     "typescript": "^4.4.0",
     "webpack": "^5.69.1",
     "webpack-cli": "^4.9.2",
-    "webpack-livereload-plugin": "^3.0.2"
+    "webpack-livereload-plugin": "^3.0.2",
+    "glob": "^8.0.3"
```

Comment:
Can we move this after `eslint-webpack-plugin` so npm/yarn won't move it in the future if/when a dev runs `yarn add anotherPackage` after scaffolding their plugin?